_drafts/oryx/oryx-versioning.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Oryx versioning
+description: Oryx framework uses semantic versioning to ensure stability
+template: concept-topic-template
+---
+
+This document describes the methods used in Oryx to deliver an advanced application development platform while maintaining stability. The goal of implementing versioning methods is to ensure that any upcoming changes are introduced in a predictable manner. This makes sure that all Oryx users are informed and adequately prepared for the release of new features and  removal of outdated ones.
+
+Stability is a critical aspect of any software framework, and it's especially important for frontend frameworks like Oryx. The stability of a framework refers to its ability to function reliably and consistently over time, despite changes in the environment or updates to its components.
+
+The following sections describe how Oryx framework achieves stability through its versioning and release processes, uses version numbers to indicate changes, and ensures backward compatibility.
+
+## Semantic versioning
+
+Oryx follows the principles of [semantic versioning](https://semver.org/).
+
+This means that, for version `x.y.z`, the following applies:
+- If critical bug fixes are released, a patch release is made by changing the `z` number. For example, `1.5.2` to `1.5.3`.
+- If new features or non-critical fixes are released, a minor release is made by changing the `y` number. For example, `1.5.2` to `15.6.0`.
+- If significant changes that may break compatibility are released, a major release is made by changing the `x` number. For example, `1.5.2` to `2.0.0`.
+
+### Release previews
+
+For those who want to know what's to come beforehand, for every major and minor release, we offer pre-release versions called Release Candidates (RC).
```

Comment:
If possible, use the closed form of compound words  - https://developers.google.com/style/word-list#closed-compounds-and-prefixes
```suggestion
For those who want to know what's to come beforehand, for every major and minor release, we offer prerelease versions called Release Candidates (RC).
```